src/typescript/sdk/tests/utils/generate-mock-data.ts

```diff
@@ -539,7 +539,7 @@ export const generateMockData = async (aptos: Aptos, publisher: Account) => {
     concatEmoji(MOCK_DATA_MARKETS_EMOJIS[2]),
     marketObjectMarketResource.metadata.marketID,
     publisher.accountAddress.toString(),
-    `${marketObjectMarketResource.metadata.marketAddress}::emojicoin_dot_fun::PeriodicState`,
+    `${publisher.accountAddress.toString()}::emojicoin_dot_fun::PeriodicState`,
```

Comment:
```suggestion
    STRUCT_STRINGS.PeriodicStateEvent,
```

BTW I added some exports for these so you don't have to write this out every time, I got tired of it, haha

Also check out `JSONTypes` and `ContractTypes` `src/typescript/sdk/src/types/contract-types.ts` and `---/json-types.ts`

Feel free to resolve this comment when you see this

Reply:
Merging per eng call discussion